build/cli.js

```diff
@@ -0,0 +1,54 @@
+const path = require("path");
+const fs = require("fs");
+
+const validCommands = [
+    "build",
+    "clean",
+    "install",
+    "installAndBuild",
+    "lint",
+    "lintfix",
+    "list",
+    "list-deps",
+    "syncdeps",
+    "test",
+];
+
+function parseCommandLine() {
+    const [, scriptPath, ...parameters] = process.argv;
+    if (parameters.includes("--help")) return { scriptPath, command: null };
+
+    const startIndex = parameters.indexOf("--start");
+    if (startIndex > 0 && startIndex + 1 >= parameters.length)
+        return { scriptPath, command: null };
+    const start =
+        startIndex !== -1 ? parameters.splice(startIndex, 2)[1] : null;
+
+    if (parameters.length > 1) return { scriptPath, command: null };
+
+    const command = parameters.length ? parameters[0] : "build";
```

Comment:
My idea was to default to `"installAndBuild"` instead, to retain the previous behaviour.

Reply:
Yes, I thought I had done that change. Let me check.

Reply:
Done now.